frontend/templates/views/partials/header.j2

```diff
@@ -87,7 +87,7 @@
     </nav>
 
     {% do doc.styles.addCssFile('/css/components/molecules/search-trigger.css') %}
-    <div id="searchTriggerOpen" class="ap-m-search-trigger" on="tap:searchLightbox" role="button"  tabindex="0">
+    <div id="searchTriggerOpen" class="ap-m-search-trigger" on="tap:searchLightbox,AMP.setState({})" role="button"  tabindex="0">
```

Comment:
`AMP.setState({})` seems unnecessary as states will get deep merged and `bigObjWithThousandKeys + {}` is still `bigObjWithThousandKeys`.

Reply:
I implemented this to force a state update on opening the search layer. This is necessary for the prefilled value of the search input field, which should contain the latest search query.